CHANGES.md

```diff
@@ -216,10 +216,10 @@ VERSIONS WILL REQUIRE PYTHON 2.7 AND ABOVE.
 ### Features
 
 -   Logging levels have been reduced.
-    -   Logging previously at the `logging.DEBUG` level is now logged at
-        the `kazoo.loggingsupport.BLATHER` level (5).
-    -   Some low-level logging previously at the `logging.INFO` level is
-        now logged at the `logging.DEBUG` level.
+-   Logging previously at the `logging.DEBUG` level is now logged at
```

Comment:
This is all that worked, I'm not exactly sure how to have that level of indentation in the docs. Didn't find any examples in this file either.